### ISSUE TITLE
Accessibility Issues

### ISSUE THREAD

**Issue by coreyshuman** [coreyshuman/coreyshuman.com#5]
- [x] Console Output in livecode does not highlight when capturing tab (could be issues on livecode as well)
- [x] Search box missing label
- [x] Images don't capture tabbing
- [x] Lightbox doesn't capture tabbing as a modal. tab order next, prev, exit. narrate image number and alt text?
- [x] Missing alt text on project related posts. is image alt text available to banner images?
- [x] Missing alt text on tags page
- [x] Is role=img correct for buttons?
- [x] Can't tab into search box results
